src/lib/viewers/BaseViewer.js

```diff
@@ -979,11 +982,23 @@ class BaseViewer extends EventEmitter {
 
         const options = boxAnnotations.getOptions && boxAnnotations.getOptions();
 
+        // Set disabled ftux cursor flag depending on if the local storage key is set
+        let isDocumentFtuxCursorDisabled = false;
+        let isImageFtuxCursorDisabled = false;
+        if (this.cache.get(DOCUMENT_FTUX_CURSOR_DISABLED_KEY)) {
```

Comment:
I could inline this into the `this.createAnnotatorOptions`, I kept it outside of that for readability purposes.